OpenRA.Game/Game.cs

```diff
@@ -655,6 +655,16 @@ static void RenderTick()
 			{
 				++RenderFrame;
 
+				// Prepare renderables (i.e. render voxels) before calling BeginFrame
+				using (new PerfSample("render_prepare"))
+				{
+					Renderer.WorldModelRenderer.BeginFrame();
+					if (worldRenderer != null && !worldRenderer.World.IsLoadingGameSave)
```

Comment:
Should mention what you're fixing here...

Reply:
Fixed.